src/cdk/testing/protractor/README.md

```diff
@@ -0,0 +1,4 @@
+### No longer publicly available
+
+This code is no longer publicly available. The protractor harness envrionment became
+deprecated in v12 and became unavailable with Angular CDK v14.
```

Comment:
```suggestion
The protractor harness environment is no longer part of the public API. It was
deprecated in v12 and removed from the API in v14.
```

Reply:
oh yeah, and also a typo! my english today 😶‍🌫️